blocks/library/more/index.js

```diff
@@ -68,33 +68,48 @@ export const settings = {
 		],
 	},
 
-	edit( { attributes, setAttributes, isSelected } ) {
-		const { customText, noTeaser } = attributes;
-
-		const toggleNoTeaser = () => setAttributes( { noTeaser: ! noTeaser } );
-		const defaultText = __( 'Read more' );
-		const value = customText !== undefined ? customText : defaultText;
-		const inputLength = value.length ? value.length + 1 : 1;
-
-		return [
-			isSelected && (
-				<InspectorControls key="inspector">
-					<ToggleControl
-						label={ __( 'Hide the teaser before the "More" tag' ) }
-						checked={ !! noTeaser }
-						onChange={ toggleNoTeaser }
+	edit: class extends Component {
+		
+		constructor() {
+			super( ...arguments );
+			this.onChangeInput = this.onChangeInput.bind( this );
+		}
+
+		onChangeInput( event ) {
+			const value = event.target.textLength === 0 ? undefined : event.target.value;
```

Comment:
I'm not familiar with `textLength`, and cannot find it documented on the [`HTMLInputElement` interface](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement). Can you refer me to more information? I might have expected `event.target.value.length` instead.

Reply:
@aduth Thanks for the heads up on the linting errors, they have been fixed.

I'm not all too familiar with `textLength` myself. Saw it was available with `event.target` and ran with it.

<img width="128" alt="screen shot 2018-03-23 at 8 46 45 pm" src="https://user-images.githubusercontent.com/52581/37817339-58445b6e-2edb-11e8-9887-1d397b12032e.png">

Looking into it further, it's apparently an SVG attribute available in the DOM via React 16: https://reactjs.org/docs/dom-elements.html. 

<img width="944" alt="screen shot 2018-03-23 at 8 54 46 pm" src="https://user-images.githubusercontent.com/52581/37817670-73d3ddd6-2edc-11e8-87a2-3e8bf6652df7.png">


> As of React 16, any standard or custom DOM attributes are fully supported

So yeah, probably best to just use `event.target.value.length` 👍 (updated PR)